packages/sandbox/src/file_watching_sandbox.ts

```diff
@@ -209,6 +227,7 @@ export class FileWatchingSandbox extends EventEmitter implements Sandbox {
    */
   stop = async () => {
     this.printer.log(`[Sandbox] Shutting down`, LogLevel.DEBUG);
+    this.functionsLogStreamer?.stopWatchingLogs();
```

Comment:
should this be `stopStreamingLogs` for naming consistency sake?

Reply:
yeah I can change both start and stop to say `stream` rather than watch.